talk/owt/sdk/base/customizedaudiocapturer.cc

```diff
@@ -131,8 +131,11 @@ bool CustomizedAudioCapturer::Playing() const {
   return false;
 }
 int32_t CustomizedAudioCapturer::StartRecording() {
-  recording_ = true;
-  const char* thread_name = "webrtc_audio_module_capture_thread";
+  {
+    rtc::CritScope lock(&crit_sect_);
```

Comment:
It looks like `crit_sect_` protects `recording_buffer_` instead of `recording_`. Perhaps it should be changed to a more meaningful name.